### PR TITLE
[FIX](complextype) fix empty quote with complex type

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes
when we using stream load to put empty quote to complex type 
which we select it will different from insert into statement 
like this:
img_v3_025n_08bc32ec-f233-4d91-b74e-34ea8022a67g

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — DISMISSED** [review]
LGTM

File: be/src/util/slice.h [hunk 1]
```diff
@@ -168,8 +168,8 @@ struct Slice {
     bool trim_quote() {
         int32_t begin = 0;
         bool change = false;
-        if (size > 2 && ((data[begin] == '"' && data[size - 1] == '"') ||
-                         (data[begin] == '\'' && data[size - 1] == '\''))) {
+        if (size >= 2 && ((data[begin] == '"' && data[size - 1] == '"') ||
```
Comment (HHoflittlefish777) [inline be/src/util/slice.h:171]: Method trim_double_quotes in slice class also need the change.

Comment (amorynan) [reply be/src/util/slice.h:171]: OK

**Comment by amorynan** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by amorynan** [comment]
run p0

**Comment by amorynan** [comment]
run external

**Comment by amorynan** [comment]
run clickbench

**Review by bobhan1 — APPROVED** [review]

**Pull request merged**